R/applications.R

```diff
@@ -218,7 +218,12 @@ showLogs <- function(appPath = getwd(), appFile = NULL, appName = NULL,
                      account = NULL, server = NULL, entries = 50, streaming = FALSE) {
 
   # determine the log target and target account info
-  target <- findDeployment(appPath, appName, account, server)
+  target <- findDeployment(
```

Comment:
I'm not seeing this as sufficient..

```
> target <- rsconnect:::findDeployment(appPath = getwd(), appName = "newshiny", account = "aronatkins", server = "shinyapps.io")
> is.null(target)
[1] FALSE
> accountDetails <- rsconnect:::accountInfo(target$account, target$server)
> client <- rsconnect:::clientForAccount(accountDetails)
> application <- rsconnect:::getAppByName(client, accountDetails, target$appName)
> is.null(application)
[1] TRUE
```

It looks like we have `target$name` but not `target$appName`...

Reply:
Using `name` instead of `appName` appears to give me a valid application.

Reply:
Here, we are using a bare deployment object, which has its name as `name`. This is different than the `deployApp` paths, which call `deploymentTarget` and rewrite the deployment `name` to `appName`.

Reply:
Sigh, I'll guess I'll try testing it then 😄